source/reviews/standard.txt

```diff
@@ -0,0 +1,90 @@
+======================
+The Standard of Review
+======================
+
+.. _pr-standard:
+
+Purpose
+-------
+
+The primary purpose of a review is to make sure that the overall health of MongoDB's
+corpus is improving over time. All of the tools and processes of review are
+designed to this end.
+
+In order to accomplish this, a series of trade-offs have to be balanced.
+
+First, authors must be able to **make progress** on their tasks. If you never
+submit an improvement to the corpus, then the corpus never improves. Also,
+if a reviewer makes it very difficult for *any* change to go in, then authors
+are disincentivized to make improvements in the future.
+
+On the other hand, it is the duty of the reviewer to make sure that each
+PR is of such a quality that the overall content health of their corpus
```

Comment:
I noticed the term "content health", but haven't seen a definition in this PR. Could this be defined somewhere so it's clear what that means?